### ISSUE TITLE
Missing fungi in plusPF and plusPFP RefSeq dbs

### ISSUE THREAD

**Issue by slw287r** [BenLangmead/aws-indexes#6]
Hi there,

I tried to use your Kraken2 plusPF db version 12/2/2020 but I can't find any fungus within the `Inspect` file as the older version 9/19/2020.

* Example of missing line
```
0.00  24687   0       K       4751            Fungi
```

<table>
<thead>
<tr>
  <th>Collection</th>
  <th>Contains</th>
  <th>Date</th>
  <th style="text-align: right">Archive size (GB)</th>
  <th style="text-align: right">Index size (GB)</th>
  <th>HTTPS URL</th>
  <th>S3 URL</th>
  <th>Inspect</th>
</tr>
</thead>
<tbody>
<tr>
	  <td>PlusPF</td>
	  <td>Standard plus protozoa &amp; fungi</td>
	  <td>12/2/2020</td>
	  <td style="text-align: right">36.9</td>
	  <td style="text-align: right">47.8</td>
	  <td><a href="https://genome-idx.s3.amazonaws.com/kraken/k2_pluspf_20201202.tar.gz">.tar.gz</a></td>
	  <td><a href="s3://genome-idx/kraken/k2_pluspf_20201202.tar.gz">.tar.gz</a></td>
	  <td><a href="https://genome-idx.s3.amazonaws.com/kraken/pluspf_20201202/inspect.txt">.txt</a></td>
</tr>
<tr>
  <td>PlusPFP</td>
  <td>Standard plus protozoa, fungi &amp; plant</td>
  <td>12/2/2020</td>
  <td style="text-align: right">70.2</td>
  <td style="text-align: right">94.3</td>
  <td><a href="https://genome-idx.s3.amazonaws.com/kraken/k2_pluspfp_20201202.tar.gz">.tar.gz</a></td>
  <td><a href="s3://genome-idx/kraken/k2_pluspfp_20201202.tar.gz">.tar.gz</a></td>
  <td><a href="https://genome-idx.s3.amazonaws.com/kraken/pluspfp_20201202/inspect.txt">.txt</a></td>
</tr>
<tr>
  <td>PlusPF</td>
  <td>Standard plus protozoa &amp; fungi</td>
  <td>9/19/2020</td>
  <td style="text-align: right">37.0</td>
  <td style="text-align: right">48.0</td>
  <td><a href="https://genome-idx.s3.amazonaws.com/kraken/k2_pluspf_20200919.tar.gz">.tar.gz</a></td>
  <td><a href="s3://genome-idx/kraken/k2_pluspf_20200919.tar.gz">.tar.gz</a></td>
  <td><a href="https://genome-idx.s3.amazonaws.com/kraken/pluspf_20200919/inspect.txt">.txt</a></td>
</tr>
<tr>
  <td>PlusPFP</td>
  <td>Standard plus protozoa, fungi &amp; plant</td>
  <td>9/19/2020</td>
  <td style="text-align: right">66.5</td>
  <td style="text-align: right">90.0</td>
  <td><a href="https://genome-idx.s3.amazonaws.com/kraken/k2_pluspfp_20200919.tar.gz">.tar.gz</a></td>
  <td><a href="s3://genome-idx/kraken/k2_pluspfp_20200919.tar.gz">.tar.gz</a></td>
  <td><a href="https://genome-idx.s3.amazonaws.com/kraken/pluspfp_20200919/inspect.txt">.txt</a></td>
</tr>
</tbody>
</table>

Best,
Richard

**Comment by BenLangmead** [comment]
You are right, Richard.  :-(  This is an error on our part.  I changed the website to omit these links until we can fix the issue.  I think we can have it fixed -- with corrected links posted -- by the end of the week.

**Comment by BenLangmead** [comment]
So far I have fixed all but the big PFP database.  That one is still building.  But I have updated links to the 5 fixed indexes, added a note to the website, and confirmed that Fungi are now included by looking at the inspect.txt files.  Will close this once I have the final index in place, but feel free to test what I've already updated.

**Comment by BenLangmead** [comment]
This is now totally fixed.  Thank you for this timely feedback!  New PF and PFP indexes available here and I've confirmed that Fungi are included: https://benlangmead.github.io/aws-indexes/k2.  New date (20210127) reflects the day I started the update process, and therefore the date when the fungi database was retrieved from refseq.  The fix will also be present in the next big release in March (at which point, all refseq databases will again be in sync w/r/t when they were downloaded).
